### PR TITLE
Separate GTFS generator into standalone service

### PR DESCRIPTION
**Pull request by frsaba**
Calls to _region/gtfs_ and _gtfs-rt_ now forward that call to the separate GTFS generator app configured with _generator_url_. The response is cached for GTFS requests within the same day.

### CODE REVIEW

File: amarillo/plugins/gtfs_export/router.py [hunk 1]
```diff
@@ -53,12 +68,25 @@ async def get_file(region_id: str, user: str = Depends(verify_admin_api_key)):
...
     elif format == 'protobuf':
-        return FileResponse(f'data/gtfs/amarillo.{region_id}.gtfsrt.pbf')
+        file_path = f'data/gtfs/amarillo.{region_id}.gtfsrt.pbf'
     else:
         message = "Specified format is not supported, i.e. neither protobuf nor json."
-        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail=message)
\ No newline at end of file
+        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail=message)
+    
+    if is_cached(file_path):
```
Comment (hbruch) [inline amarillo/plugins/gtfs_export/router.py]: GTFS-RT stand for GFTFS-Realtime. This feed should have a max time to live of 60 secs. I.e. it must not be cached for a whole days

Comment (frsaba) [reply amarillo/plugins/gtfs_export/router.py]: Fixed, I have removed caching for GTFS-RT

Comment (hbruch) [reply amarillo/plugins/gtfs_export/router.py]: This is a behavior change in contrast to before. Before, it was re-generated once per minute (which would correspond to caching of one minute). As this file can be requested very often, it should be cached for a minute.

Comment (frsaba) [reply amarillo/plugins/gtfs_export/router.py]: Okay, I've added caching of GTFS-RT files for 1 minute.

**Pull request merged**